src/qrcode/QRScanner.tsx

```diff
@@ -69,6 +69,7 @@ export default function QRScanner({ onBarCodeDetected }: QRScannerProps) {
   }
 
   const submitModal = () => {
+    Keyboard.dismiss()
```

Comment:
Is this change related to the freeze feature?

Reply:
This is there was a failing e2e test in CI where the keyboard was not dismissed. I was unable to reproduce locally and this modal is only used in e2e tests so it seems fine to have this dismiss here.